src/bitmessagekivy/baseclass/allmail_widgets.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=too-few-public-methods
+
+"""
+Widgets using in allmail are here.
+"""
+
+
+class HelperAllMail(object):
```

Comment:
maybe no need for a class? just a single module function, or even put it into common

Reply:
I have moved `msg_content_length()` method inside `allmail.py` so not needed `allmail_widgets.py`.